Makefile.arm

```diff
@@ -1,39 +1,44 @@
-CROSS=arm-none-eabi-
+CROSS=arm-linux-gnueabi-
```

Comment:
The `arm-none-eabi-` is intentional for bare metal ARM boards without Linux, for `arm-linux-gnueabi-` you would probably want to use the normal makefile on an ARM machine, or with a cross-compiler specified when building on a non-ARM machine. If there is some reason that doesn't work, then it might make sense to either make a new Makefile for this case, or add some configurable options to the main one (and document them).